bin/build_cmd.mli

```diff
@@ -12,3 +12,5 @@ val runtest_term : unit Term.t
 val build : unit Cmd.t
 
 val fmt : unit Cmd.t
+
+val doc : unit Cmd.t
```

Comment:
I don't think this is the correct place for this. Rather than putting it here, could you put it in a file called doc.ml and place that file in bin/ocaml next to all the other ocaml subcommands?